news/_posts/releases/2019-07-10-prestashop-1-7-6-0-available.md

```diff
@@ -0,0 +1,194 @@
+- In the meantime, we also fixed a bug on carrier price calculation. Price is now calculated on the real order price and not the one including after a discount is applied
+- DNI fields is now forced only for countries which require it
+- Fixed the minimum order in the second currency bug - fixed by the [community](https://github.com/PrestaShop/PrestaShop/issues/9665)
+
+**Core**
+- Themes are not reset anymore when already active (also prevent hooks to be reset) - fixed in the Autoupgrade module
+
+
+## The technical side
+
+### Technical improvements
+
+- Nightly builds and nightly board are [now available](https://nightly.prestashop.com/).
+- Translation Manager of Back Office can now manage legacy translations of modules as well as symfony modules
+- Added support for Doctrine Entities in modules (both Front and Back-office)
+- Price computation is now covered by Integration tests powered by Behat (human-readable tests)
+- New mail template system based on Twig, along with a new generation system which avoids downloading all the mail templates; will allow easier customization in the future versions
+- New CLDR standard implementation: improved localization of prices and amounts, will allow creating custom currencies in future versions
+- Support for multiple grids on the same page
+- Performance:
+  - Improved employee authorization process
+  - Cache introduced when loading modules
+  - The number of SQL requests to get the employee authorizations has been significantly reduced
+- Web services:
+  - Product customization is now reachable
+  - Product combination information in Pack content is now available
+
+
+
+### New hooks
+- displayProductActions (Product page, close to the “Add to cart” button)
+- displayPersonalInformationTop (Checkout funnel, before the customer login form)
+- additionalCustomerAddressFields (In CustomerAddressFormatter)
+- actionProductFlagsModifier (Allows to customize product flags)
+- Form hooks
+- Kpi hooks
+- Mail generator hooks:
+  - actionListMailThemes
+  - actionGetMailThemeFolder
+  - actionBuildLayoutVariables
+  - actionGetMailLayoutTransformations
+  
+See all hooks in the [DevDocs](https://devdocs.prestashop.com/1.7/modules/concepts/hooks/list-of-hooks/)
+
+
+### Improved hooks
+
+- actionProductAdd now receives id_product_old when duplicating a product
+
+
+
+### Breaking or risky changes
+
+- The new CLDR implementation replaces the old one. Some breaking changes were needed:
+  - The Currency ObjectModel is now multilingual
+  - The IcanBoogie/CLDR library has been removed
+  - Calls to Tools::getCldr() throw a PrestaShopException
+  - The namespace “PrestaShop\PrestaShop\Core\Cldr” has been removed
+  - The CLDR files in the “/translations” directory have been removed
+- Import:
+  - You no longer can just add categories to current categories with import to a force ID. You will have to set them all and the first will become the default_category.)
```

Comment:
`You no longer can` => `You can no longer`
And there is a `)` at the end of the line, but no `(` anywhere